test-toolkit/docker/1.6.0/Dockerfile.mxnet.cpu

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    software-properties-common \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    openssh-client \
+    openssh-server \
+    vim \
```

Comment:
vim is needed for MXNet 😮

Reply:
nah, but it doesn't take that much and useful to have for manual testing :)